modules/apis/src/main/java/com/axway/apim/apiimport/actions/RepublishToUpdateAPI.java

```diff
@@ -16,6 +22,21 @@ public void execute(APIChangeState changes) throws AppException {
 		
 		API actualAPI = changes.getActualAPI();
 		
+		Mode clientAppsMode = CoreParameters.getInstance().getClientAppsMode();
+		Mode clientOrgsMode =  CoreParameters.getInstance().getClientOrgsMode();
+		// Get existing Orgs and Apps, as they will be lost when the API gets unpublished
+		if(clientAppsMode==Mode.add && actualAPI.getApplications()!=null && changes.getDesiredAPI().getApplications()!=null) { 
```

Comment:
I believe in the scenario described in #117 , changes.getDesiredAPI().getApplications() must return null, that's why to restore previous client App subscriptions, at least 1 clientApp must be set in config file